docs/developer-guide/api/concrete.ml.onnx.ops_impl.md

```diff
@@ -1142,7 +1178,48 @@ See https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Transpose-13
 
 ______________________________________________________________________
 
-<a href="../../../src/concrete/ml/onnx/ops_impl.py#L1206"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+<a href="../../../src/concrete/ml/onnx/ops_impl.py#L1143"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+## <kbd>function</kbd> `numpy_conv`
+
+```python
+numpy_conv(
+    x: ndarray,
+    w: ndarray,
+    b: Optional[ndarray] = None,
+    dilations: Tuple[int, ],
+    group: int = 1,
+    kernel_shape: Tuple[int, ],
+    pads: Tuple[int, ],
+    strides: Tuple[int, ]
+) → Tuple[ndarray]
+```
+
+Compute N-D convolution using Torch.
+
+Currently supports 2d convolution with torch semantics. This function is also ONNX compatible.
+
+See: https://github.com/onnx/onnx/blob/main/docs/Operators.md#Conv
+
+**Args:**
+
+- <b>`x`</b> (numpy.ndarray):  input data (many dtypes are supported). Shape is N x C x H x W for 2d
+- <b>`w`</b> (numpy.ndarray):  weights tensor. Shape is (O x I x Kh x Kw) for 2d
+- <b>`b`</b> (Optional\[numpy.ndarray\]):  bias tensor, Shape is (O,). Default to None.
+- <b>`dilations`</b> (Tuple\[int, ...\]):  dilation of the kernel, default 1 on all dimensions.
+- <b>`group`</b> (int):  number of convolution groups, can be 1 or a multiple of both (C,) and (O,), so  that I = C / group. Default to 1.
+- <b>`kernel_shape`</b> (Tuple\[int, ...\]):  shape of the kernel. Should have 2 elements for 2d conv
+- <b>`pads`</b> (Tuple\[int, ...\]):  padding in ONNX format (begin, end) on each axis
+- <b>`strides`</b> (Tuple\[int, ...\]):  stride of the convolution on each axis
+
```

Comment:
Do we use full stops as the standard for args/returns? Add them here so the file is consistent.

Reply:
it will be a some work to make these uniform (there are 200 occurences only for Returns), I'll make an issue on it